api/app/db/crud/c_haines.py

```diff
@@ -123,13 +112,13 @@ def get_prediction_geojson(session: Session,
                            model: ModelEnum,
                            model_run_timestamp: datetime,
                            prediction_timestamp: datetime):
-    """ Get the geojson for a particular prediction """
+    """ Get the geojson for a particular prediction, in WGS84 """
     query = text("""select json_build_object(
         'type', 'FeatureCollection',
         'features', json_agg(ST_AsGeoJSON(t.*)::json)
     )
         from (
-        select geom, c_haines_index from c_haines_polygons
+        select ST_Transform(geom, 'epsg:4269', 'epsg:4326'), c_haines_index from c_haines_polygons
```

Comment:
what transformation is going on here? Why is the coordinate system being changed?

Reply:
Nvm - found the answer to my question in another file. But I'd recommend adding a comment here to explain why the coordinate system needs to be transformed.

Reply:
I here you - but this code is about to get deleted, so not going to bother. Answer is - in the DB it's NAD83 - but KML and GeoJSON are WGS84.